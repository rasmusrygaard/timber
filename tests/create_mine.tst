create foo 123
create / 123
create // 123
create /// 123
create //// 123
create /foo/ 123
create /foo 123
create /foo/bar/ 123
create /foo/bar// 123
create /!!! 123
create /foo! 123
create /foo_bar_baz 123
create /foo_123_bar 123
create /foo____ 123
