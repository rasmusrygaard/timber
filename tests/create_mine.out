MALFORMED KEY
MALFORMED KEY
MALFORMED KEY
MALFORMED KEY
MALFORMED KEY
MALFORMED KEY
CREATED
MALFORMED KEY
MALFORMED KEY
MALFORMED KEY
MALFORMED KEY
CREATED
CREATED
CREATED
